modules/light-clients/08-wasm/types/wasm.go

```diff
@@ -0,0 +1,45 @@
+package types
+
+import (
+	"bytes"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// GetCodeHashes returns all the code hashes stored.
+func GetCodeHashes(ctx sdk.Context, cdc codec.BinaryCodec) CodeHashes {
+	wasmStoreKey := GetWasmStoreKey(cdc)
+	store := ctx.KVStore(wasmStoreKey)
+	bz := store.Get([]byte(KeyCodeHashes))
+	if len(bz) == 0 {
+		return CodeHashes{}
+	}
+	var hashes CodeHashes
+	cdc.MustUnmarshal(bz, &hashes)
+	return hashes
+}
+
+// AddCodeHash adds a new code hash to the list of stored code hashes.
+func AddCodeHash(ctx sdk.Context, cdc codec.BinaryCodec, codeHash []byte) {
+	codeHashes := GetCodeHashes(ctx, cdc)
+	codeHashes.Hashes = append(codeHashes.Hashes, codeHash)
```

Comment:
not the most efficient sorting/searching, but I guess when you only have 5 code hashes, it isn't an issue